podcastparser.py

```diff
@@ -674,8 +674,18 @@ def validate_episode(self):
         if len(entry['chapters']) == 0:
             del entry['chapters']
 
-        if 'description_html' in entry and not entry['description']:
-            entry['description'] = remove_html_tags(entry['description_html'])
+        # Ensures `description` and `description_html` are plain text and html if provided
+        if 'description' in entry and is_html(entry['description']):
+            if 'description_html' in entry:
+                if entry['description'] == entry['description_html']:
+                    entry['description'] = ''
+                else:
+                    logger.warning(
```

Comment:
Since I expect this to be an rate situation possibly pointing to a bug in our code (erroneous HTML detection) that leads to data loss, my gut says there should be some visibility to it either via log or code comment.  I added feed url and episode title to the log message, though I would also be fine removing the logging all together.